warehouse/migrations/versions/75ba94852cd1_make_pendingoidcpublisher_added_by_id_.py

```diff
@@ -0,0 +1,42 @@
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Make PendingOIDCPublisher.added_by_id non-nullable
+
+Revision ID: 75ba94852cd1
+Revises: f7cd7a943caa
+Create Date: 2023-04-14 18:21:38.683694
+"""
+
+from alembic import op
+from sqlalchemy.dialects import postgresql
+
+revision = "75ba94852cd1"
+down_revision = "f7cd7a943caa"
+
+
+def upgrade():
+    op.alter_column(
+        "pending_oidc_publishers",
+        "added_by_id",
+        existing_type=postgresql.UUID(),
+        nullable=False,
+    )
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
**thought:** ‏This kind of `ALTER COLUMN` statement will check for validity immediately - is there some other process that guarantees all records are non-null before this migration runs?

Reply:
Great point: there is not, however it doesn't seem like it was previously possible to add a row with a null column here. I need to verify this in the database before merging.